theme.js

```diff
@@ -0,0 +1,41 @@
+//Prepares the document (check for jQuery). Remember to load jQuery in the html file BEFORE running any scripts with jQuery.
```

Comment:
```
var icon = document.getElementById("icon");

icon.onclick = function(){
    document.body.classList.toggle("dark-theme");

    if(document.body.classList.contains("dark-theme")){
        icon.src="/grossmont/computer-science-and-cybersecurity-organization/media/sun.png";
    }
    else{
        icon.src="/grossmont/computer-science-and-cybersecurity-organization/media/moon.png";
    }
}
```
So I'm not sure how this works, since there wasn't any files that actually contained dark-theme

Reply:
Ohh... That makes sense. I assumed since it was in the style.css, and most of the html were using it as a stylesheet it would interact with it through that.